_po/ja/how-to/laravel.po

```diff
@@ -493,11 +724,20 @@ msgid ""
 "\t</div>\n"
 "```"
 msgstr ""
+"変更後:\n"
+"```php\n"
+"<!-- Line 25 -->\n"
+"\t<div class=\"w-full lg:max-w-6xl mt-6 px-6 py-4 bg-white shadow-md overflow-hi"
+"dden sm:rounded-lg\">\n"
+"\t    {{ $slot }}\n"
+"\t</div>\n"
+"```"
```

Comment:
Probably, this is needless.
Because this test is same as before translation text.